language-server/modules/langserver-core/src/test/resources/performance/sources/main.bal

```diff
@@ -0,0 +1,99 @@
+import wso2/nballerina.types as t;
+import wso2/nballerina.bir;
+import wso2/nballerina.front;
+import wso2/nballerina.nback;
+import wso2/nballerina.err;
+
+import ballerina/io;
+import ballerina/file;
```

Comment:
let's keep this file as a file with diagnostics and this source is good. If we need un resolved modules, let's use some other values other than the existing io, file and wso2 based imports.